RHEL/7/input/profiles/ospp-rhel7-server.xml

```diff
@@ -0,0 +1,135 @@
+<Profile id="ospp-rhel7-server" extends="common">
```

Comment:
On 8/27/15 4:39 PM, Robin Price II wrote:
>
> In RHEL/7/input/profiles/ospp-rhel7-server.xml 
> <https://github.com/OpenSCAP/scap-security-guide/pull/656#discussion_r38143206>:
>
> > @@ -0,0 +1,135 @@
> > +<Profile id="ospp-rhel7-server" extends="common">
>
> File name and Profile id?
>
There's no standard mandating the alignment of profile names to 
filenames, but this does make things easier for developers. So yes, 
please :)